src/config/dark-sites.config

```diff
@@ -8,7 +8,8 @@
 2bqueue.info
 314n.org
 5stardata.info
-9anime.id
+9anime.gs
+9anime.pl
```

Comment:
So many changes for 9anime the last few year... I'm tempted to exclude them from this list as I'm not keeping up with this.